### PR TITLE
Add a note for Eclipse users and local builds.

### PR DESCRIPTION
**Pull request by deflaux**
@dloy please take a look

@jean-philippe-martin it might be helpful if you add similar instructions for intelliJ?

### CODE REVIEW

**Comment by jean-philippe-martin** [comment]
@deflaux, can do.

**Comment by jean-philippe-martin** [comment]
I find those instructions a little bit confusing, because the Eclipse instructions say "First update the dependencies in your poms to point to the SNAPSHOT versions as mentioned above." but it's not clear that one should only do that if they want to work on multiple repos at once. The section's title is just "Building this project".

**Comment by deflaux** [comment]
@jean-philippe-martin ptal

File: README.md [hunk 1]
```diff
@@ -42,11 +42,15 @@ Use a recent version of [Apache Maven](http://maven.apache.org/download.cgi) (e.
...
     mvn package
 
-If you wish to modify this code while also modifying a project that depends upon it (e.g., [dataflow-java](https://github.com/googlegenomics/dataflow-java), just
+If you wish to modify this code while also modifying a project that depends upon it (e.g., [dataflow-java](https://github.com/googlegenomics/dataflow-java)), just
 
   1. Build and install the jar to your local maven repository via `mvn install`.
   2. Then update the pom.xml of the dependent project to temporarily depend upon the particular SNAPSHOT version of utils-java now installed in your local maven repository.
-
+  3. Eclipse users: 
+    * First update the dependencies in your poms to point to the SNAPSHOT versions as mentioned above.
```
Comment (dionloy) [inline README.md]: I'm not familiar with Maven other than 'mvn install' =). Do you have a pointer or clarification on the SNAPSHOT version stuff?  (just append -SNAPSHOT to the current version, or is something else required here?).

Comment (jean-philippe-martin) [inline README.md]: Isn't (3.a) redundant with point (2)? I think we can just remove it. Other than that, the indentation totally cleared up the confusion, thank you!

**Comment by deflaux** [comment]
How's this version?  The basic idea is that maven reads from the local maven repository.  Eclipse will do its own build, but it can figure out the dependencies from the pom file.  I tend to use both together, as a sanity check.

Please feel free to tweak the wording so that it is more clear.  Also, we can dig up links to the maven documentation and eclipse documentation.  None of this is specific to genomics, but it is really helpful to give people the cookbook summary right here!

**Comment by dionloy** [comment]
LGTM Thanks!

**Pull request merged**